en/flight_controller/durandal.md

```diff
@@ -0,0 +1,365 @@
+  - Max input voltage: 6V
+  - Max current sensing: 120A
+  - USB Power Input: 4.75~5.25V
+  - Servo Rail Input: 0~36V
+
+- Mechanical Data:
+  - Dimensions: 80x45x20.5mm
+  - Weight: 68.8g
+  
+- Other Characteristics:
+  - Operating temperature: ~40~85C
+  - Storage temperature: -40~85C
+  - CE
+  - FCC
+  - RoHS compliant (lead-free)
+
+For more information see: [Durandal Technical Data Sheet](http://www.holybro.com/manual/Durandal_technical_data_sheet.pdf).
+
+## Purchase {#purchase}
+
+Order from [Holybro](https://shop.holybro.com/durandalbeta_p1189.html).
+
+
+## Connectors
+
+The connectors are shown in the [pinouts section](#pinouts) below.
+
+
+## Dimensions
+
+All dimensions are in millimeters.
+
+![Durandal Dimensions](../../assets/flight_controller/durandal/durandal_dimensions.jpg)
+
+<!-- 
+## Voltage Ratings
+
+*Pixhawk 4* can be triple-redundant on the power supply if three power sources are supplied. The three power rails are: **POWER1**, **POWER2** and **USB**.
+
+> **Note** The output power rails **FMU PWM OUT** and **I/O PWM OUT** (0V to 36V) do not power the flight controller board (and are not powered by it). You must supply power to one of **POWER1**, **POWER2** or **USB** or the board will be unpowered.
+
+**Normal Operation Maximum Ratings**
+
+Under these conditions all power sources will be used in this order to power the system:
+1. **POWER1** and **POWER2** inputs (4.9V to 5.5V)
+1. **USB** input (4.75V to 5.25V)
+-->
+
+<!-- 
+**Absolute Maximum Ratings**
+
+Under these conditions the system will not draw any power (will not be operational), but will remain intact.
+1. **POWER1** and **POWER2** inputs (operational range 4.1V to 5.7V, 0V to 10V undamaged)
+1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
+1. Servo input: VDD_SERVO pin of **FMU PWM OUT** and **I/O PWM OUT** (0V to 42V undamaged)
+
+-->
+
+## Assembly/Setup
+
+The [Durandal Wiring Quick Start](../assembly/quick_start_durandal.md) provides instructions on how to assemble required/important peripherals including GPS, Power Management Board etc.
+
+
+## Building Firmware
+
+> **Tip** Most users will not need to build this firmware!
+  It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
+
+To [build PX4](https://dev.px4.io/master/en/setup/building_px4.html) for this target:
+```
+make holybro_durandal-v1_default
+```
+
+
+## Debug Port
+
+The system's serial console and SWD interface runs on the [Debug port](#debug_port).
+
+The port has a  standard serial pinout and can be connected to a standard FTDI cable (3.3V, but it's 5V tolerant) or a [Dronecode probe](https://kb.zubax.com/display/MAINKB/Dronecode+Probe+documentation).
+The pinout uses the standard Dronecode debug connector pinout.
+Please refer to the [wiring](https://dev.px4.io/master/en/debug/system_console.html) page for details of how to wire up this port.
+
+<!-- I/O port - how do we access? -->
```

Comment:
So debug port is exposed but I assume this is for debugging FMU, not IO. IS there an IO DEBUG port, and if so do we need to mention it.

Reply:
It's not exposed and I'm not sure if you can access internally.